python/cuml/test/test_random_forest.py

```diff
@@ -596,3 +596,29 @@ def test_rf_memory_leakage(fil_sparse_format, column_info, nrows):
         handle.sync()
         delta_mem = free_mem - cuda.current_context().get_memory_info()[0]
         assert delta_mem == 0
+
+
+@pytest.mark.parametrize('datatype', [np.float32])
+@pytest.mark.parametrize('max_features', [1.0, 'auto', 'log2', 'sqrt'])
+@pytest.mark.parametrize('max_depth', [10, 13, 16])
+@pytest.mark.parametrize('n_estimators', [10, 20, 100])
+@pytest.mark.parametrize('n_bins', [8, 9, 10])
+def test_create_classification_model(datatype,
```

Comment:
To test this exact issue, it seems like we should test  creating an empty model (model2), and calling:

```
model2.set_params(model1._get_params())
# assert some important features are equivalent here
```

Also, since you aren't doing a fit, no need to create a dataset.